docs/stdlib/datetime.rst

```diff
@@ -212,19 +208,20 @@ EdgeDB stores and outputs timezone-aware values in UTC.
         "cal::to_local_datetime" function provides custom formatting
         options.
 
-    All ``datetime`` values are restricted to the range from year 1 to 9999.
+    All :eql:type:`cal::local_datetime` values are restricted to a range of
+    year 1 to 9999.
 
-    See functions :eql:func:`datetime_get`, :eql:func:`cal::to_local_datetime`,
-    and :eql:func:`to_str` for more ways of working with
-    :eql:type:`cal::local_datetime`.
+    For more information regarding interacting with this type, see
+    :eql:func:`datetime_get`, :eql:func:`cal::to_local_datetime` and
```

Comment:
```suggestion
    :eql:func:`datetime_get`, :eql:func:`cal::to_local_datetime`, and
```

Restoring the Oxford comma, just so we don't unintentionally introduce any ambiguity by applying it inconsistently. I recommend applying this globally to all series.